pkg/simple/client/k8s/options.go

```diff
@@ -44,12 +47,18 @@ type KubernetesOptions struct {
 }
 
 // NewKubernetesOptions returns a `zero` instance
-func NewKubernetesOptions() *KubernetesOptions {
-	return &KubernetesOptions{
-		KubeConfig: "",
-		QPS:        1e6,
-		Burst:      1e6,
+func NewKubernetesOptions() (option *KubernetesOptions) {
+	option = &KubernetesOptions{
+		QPS:   1e6,
+		Burst: 1e6,
+	}
+
+	// make it be easier for those who wants to run api-server locally
+	userHomeConfig := path.Join(homedir.HomeDir(), ".kube/config")
+	if _, err := os.Stat(userHomeConfig); !os.IsNotExist(err) {
```

Comment:
I'd prefer to judge if err==nil. Why do you only handle the `NotExist` err?

Reply:
Excellent suggestion!